src/Bernard/Driver/FlatFileDriver.php

```diff
@@ -18,13 +18,20 @@ class FlatFileDriver implements Driver
     private $baseDirectory;
 
     /**
+     * @var int
+     */
+    private $permissions;
+
+    /**
      * Constructor
      *
      * @param string $baseDirectory The base directory
+     * @param int    $permissions   Permissions to create the file with.
      */
-    public function __construct($baseDirectory)
+    public function __construct($baseDirectory, $permissions = 0)
```

Comment:
I think we should set an acceptable default, also i think Doctrine does this for sqlite databases.